server/controllers/auth/signupByMobile.js

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable camelcase */
```

Comment:
we don't need this rout, we already handle this in login and we font need button in the signup page